test/functional/apps/dashboard/dashboard_state.ts

```diff
@@ -66,33 +66,28 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.dashboard.clickNewDashboard();
       await PageObjects.timePicker.setHistoricalDataRange();
 
-      const visName = await PageObjects.visChart.getExpectedValue(
-        AREA_CHART_VIS_NAME,
-        `${AREA_CHART_VIS_NAME} - new charts library`
-      );
+      const visName = AREA_CHART_VIS_NAME;
       await dashboardAddPanel.addVisualization(visName);
-      const dashboarName = await PageObjects.visChart.getExpectedValue(
-        'Overridden colors',
-        'Overridden colors - new charts library'
-      );
+      const dashboarName = 'Overridden colors - new charts library';
```

Comment:
```suggestion
      const dashboardName = 'Overridden colors - new charts library';
```
nit: that's just a typo in spelling, feel free to ignore this comment.